spec/connectors_sdk/sharepoint/http_call_wrapper_spec.rb

```diff
@@ -24,7 +24,10 @@
   end
 
   let(:params) do
-    { 'access_token' => 'something' }
+    {
+      'access_token' => 'something',
+      'index_permissions' => false
```

Comment:
`false` is the default value in your code. can we switch this one to `true` and then assert that `ConnectorsSdk::Office365::Config.index_permissions` is `true` ?